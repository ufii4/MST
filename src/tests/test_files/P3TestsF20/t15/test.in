n 15
i 0;1;1
i 0;2;1
i 1;3;1
i 1;4;1
i 2;5;1
i 2;6;1
i 3;7;1
i 3;8;1
i 4;9;1
i 4;10;1
i 5;11;1
i 5;12;1
i 6;13;1
i 6;14;1
edge_count
mst
exit
