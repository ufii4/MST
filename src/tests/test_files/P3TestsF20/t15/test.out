success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
edge count is 14
mst 14.00
